### PR TITLE
Remove deprecated `validate_inputs_base`

### PR DESCRIPTION
**Pull request by mbercx**
In https://github.com/aiidateam/aiida-quantumespresso/commit/a389629387b74805ffe2f4d6515ac05b8f62b4d5, the validation of the `parent_folder` was adapted to no longer rely on splitting up the validation of the top-level inputs in `validate_inputs` and `validate_inputs_base`. Instead, a wrapping work chain can just exclude the `parent_folder` when exposing the inputs of e.g. a `PwBaseWorkChain` whose `parent_folder` will be obtained at run-time.

Here we remove the line adapting the validator of the `PwCalculation` in the `nscf` namespace of the `Wannier90WorkChain` to `validate_inputs_base`, and exclude `pw.parent_folder` from this namespace instead.

### CODE REVIEW

**Comment by mbercx** [comment]
@qiaojunfeng this PR is blocked until the commit described above is released, but already wanted to give you a heads up.

**Comment by qiaojunfeng** [comment]
Thanks, Marnik! The commit message https://github.com/aiidateam/aiida-quantumespresso/commit/a389629387b74805ffe2f4d6515ac05b8f62b4d5 is very good and clear^^

> this PR is blocked until the commit described above is released

Do you have any plan of when to release a new version of aiida-qe? Once that it done I can safely merge this

**Review by qiaojunfeng — APPROVED** [review]

**Comment by mbercx** [comment]
@qiaojunfeng I've updated the `aiida-quantumespresso` dependency to `4.4.0` here, since the change in validation described above is made there.

**Comment by mbercx** [comment]
@qiaojunfeng can I go ahead and merge this?

**Review by qiaojunfeng — APPROVED** [review]

**Comment by qiaojunfeng** [comment]
> @qiaojunfeng can I go ahead and merge this?

Sorry for the delay, just merged it!

**Pull request merged**
